plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -310,19 +311,23 @@ private static String buildSplitPath(String tableLocation, AddFileEntry addActio
     {
         // paths are relative to the table location and are RFC 2396 URIs
         // https://github.com/delta-io/delta/blob/master/PROTOCOL.md#add-file-and-remove-file
-        URI uri = URI.create(addAction.getPath());
-        String path = uri.getPath();
+        String path = addAction.getPath();
```

Comment:
We were using previously the path obtained from the `URI`.
Your change doesn't make use anymore of the URI parsed path when the path is not absolute.

Reply:
Do note the failing tests

Reply:
corrected it to use `uri.getPath()`